Managers/SceneManager.gd

```diff
@@ -82,27 +82,28 @@ func on_event_win() -> void:
 func save_scene_data() -> void:
 	var packed_scene: PackedScene = PackedScene.new()
 	packed_scene.pack(current_scene)
-	ResourceSaver.save(packed_scene, "user://data/current_scene.tscn")
+	ResourceSaver.save(packed_scene, "user://save/current_scene.tscn")
 	
 	var save_file: ConfigFile = SaveManager.save_file
 	save_file.set_value("SceneManager", "current_event", current_event)
 	
-	var error: Error = save_file.save("user://data/save_data.ini")
+	var error: Error = save_file.save("user://save/save_data.ini")
 	if error:
-		print("Error saving player data: ", error)
+		push_error("Error saving player data: ", error)
 
 func load_scene_data() -> void:
 	call_deferred("_deferred_load_current_scene_from_data")
 
 func _deferred_load_current_scene_from_data() -> void:
 	var save_file: ConfigFile = SaveManager.load_save_file()
-	assert(save_file != null, "Attempting to load a scene with no config file. How did you get here???")
+	if save_file == null:
+		push_error("Attempting to load a scene with no config file. How did you get here???")
```

Comment:
I'm nitpicking, but shouldn't it be "Attempting to load a scene with no **save** file [...]" ?